service/src/main/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCache.java

```diff
@@ -67,6 +88,11 @@ public PolicyEvaluationResult get(final PolicyEvaluationRequestCacheKey key) {
         } catch (IOException e) {
             throw new IllegalStateException("Failed to read policy evaluation result as JSON.", e);
         }
+        values.remove(values.size() - 1);
+
+        String zone = key.getZoneId();
+        values.addAll(multiGet(cachedResult.getResolvedResourceUris().stream()
+                .map(resourceUri -> resourceKey(zone, resourceUri)).collect(Collectors.toList())));
```

Comment:
I take it back.  Resolved UIRs could not be part of the eval result key as they are not known at the time of the request.